subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -621,6 +641,9 @@ static void lp_comm_rx_decode(struct ll_conn *conn, struct proc_ctx *ctx, struct
 	case PDU_DATA_LLCTRL_TYPE_REJECT_EXT_IND:
 		llcp_pdu_decode_reject_ext_ind(ctx, pdu);
 		break;
+	case PDU_DATA_LLCTRL_TYPE_REJECT_IND:
+		/* Empty on purpose, as we don't care about the PDU content, we'll disconnect */
```

Comment:
why should we disconnect on receiving a REJECT_IND?

Reply:
Argh, this should be part of PR44550, but a REJECT_IND here is not warranted here,  only REJECT_EXT in case we're running CTE procedure.

Reply:
Ahh, sorry I'm confusing myself - I have the PR44550 commit here as well. 
So - forget me leading comment and consider the trailing one only;-)